### PR TITLE
Document "nodes" in navigation-parameters-reference.md

### PR DESCRIPTION
**Pull request by alexandra-simeonova**
**Description**

Changes proposed in this pull request:

- Document "nodes" in navigation-parameters-reference.md

**Related issue(s)**
#3467

### CODE REVIEW

File: docs/navigation-parameters-reference.md [hunk 1]
```diff
@@ -185,6 +185,28 @@ Check our [Advanced Scenarios](advanced-scenarios.md) page for an example.
 - **type**: function
 - **description**: allows you to invoke and execute a specific function on the global level when a request to navigate to the node occurs. The function receives two node objects as input parameters: the previous node and current node, as described in the configuration.
 
+### nodes
+- **type**: array
```
[the remark this answers is not in the document]
Comment (alexandra-simeonova) [reply docs/navigation-parameters-reference.md:189]: Yes, you're right. I'm missing some technical details. Is it supposed to be an `Object`? Maybe @JohannesDoberer knows more. Also, the behavior we tested on Fiddle - someone deletes the root node (?) and what should be the result? Maybe we can add a note about it.

[the remark this answers is not in the document]
Comment (alexandra-simeonova) [reply docs/navigation-parameters-reference.md]: Update: changes were made in the latest commit.

File: docs/navigation-parameters-reference.md [hunk 2]
```diff
@@ -185,6 +185,48 @@ Check our [Advanced Scenarios](advanced-scenarios.md) page for an example.
 - **type**: function
 - **description**: allows you to invoke and execute a specific function on the global level when a request to navigate to the node occurs. The function receives two node objects as input parameters: the previous node and current node, as described in the configuration.
 
+### nodes
+- **type**: array | object
+- **description**: You can define navigation nodes using the `nodes:` attribute. First-level node are also referred to as "root nodes". Second-level notes can be defined inside a [children](#children) array below the root node. 
+
+Instead of an array, `nodes:` can also be defined as a single node object which then serves as the root node.
```
Comment (ndricimrr) [inline docs/navigation-parameters-reference.md:192]:
```suggestion
In addition to an array, `nodes:` can also be defined as a single node object which then serves as the root node. 
```

File: docs/navigation-parameters-reference.md [hunk 3]
```diff
@@ -185,6 +185,48 @@ Check our [Advanced Scenarios](advanced-scenarios.md) page for an example.
 - **type**: function
 - **description**: allows you to invoke and execute a specific function on the global level when a request to navigate to the node occurs. The function receives two node objects as input parameters: the previous node and current node, as described in the configuration.
 
+### nodes
+- **type**: array | object
+- **description**: You can define navigation nodes using the `nodes:` attribute. First-level node are also referred to as "root nodes". Second-level notes can be defined inside a [children](#children) array below the root node.
```
Comment (ndricimrr) [inline docs/navigation-parameters-reference.md:190]:
```suggestion
- **description**: You can define navigation nodes using the `nodes:` attribute. First-level nodes are also referred to as "root nodes". Second-level nodes can be defined inside a [children](#children) array below the root node. 
```

**Review by ndricimrr — APPROVED** [review]
Looks good aside from some typos

**Pull request merged**